### ISSUE TITLE
Rebuild all frameworks with BUILD_LIBRARY_FOR_DISTRIBUTION = YES;

### ISSUE THREAD

**Issue by RomanPodymov** [Paytm-Payments/Paytm_iOS_App_Kit#45]
Hello.
Found out that you don't use `BUILD_LIBRARY_FOR_DISTRIBUTION` to build  Swift frameworks in this repository. You should enable this option to fix all the issues like #40 and #43.

**Comment by amitmeh10** [comment]
its not working..

**Comment by amitmeh10** [comment]
Any one have solution for that for Swift Compiler 5.1.2 ?

**Comment by RomanPodymov** [comment]
Hello @amitmeh10 
You cannot rebuild the framework on your side with `BUILD_LIBRARY_FOR_DISTRIBUTION` enabled if you don't have access to source code. You should wait for the release that supports this feature.

**Comment by amitmeh10** [comment]
i solved it. just change Do not Embed to Embed & sign... and its work fine...

**Comment by RomanPodymov** [comment]
Hello @amitmeh10 
Are you sure that your project is still runnable if you don't embed the framework that has `BUILD_LIBRARY_FOR_DISTRIBUTION = NO`?

**Comment by amitmeh10** [comment]
yes i m not change NO to YES for BUILD_LIBRARY_FOR_DISTRIBUTION.
But still my code is runnable . and if i change it YES than its shows me error to build bridging header file.

**Comment by amitmeh10** [comment]
> Changing to below configuration under the general tab will work.
> 
> ![Screenshot 2020-01-09 at 11 39 32 AM](https://user-images.githubusercontent.com/13044958/72042650-22b60b00-32d5-11ea-8364-453b04111dd0.png)
> 
> Also, note for release build you need to add this script under build scripts
> 
> if [ "${CONFIGURATION}" = "Release" ]; then
> APP_PATH="${TARGET_BUILD_DIR}/${WRAPPER_NAME}"
> 
> # This script loops through the frameworks embedded in the application and
> # removes unused architectures.
> find "$APP_PATH" -name '*.framework' -type d | while read -r FRAMEWORK
> do
> FRAMEWORK_EXECUTABLE_NAME=$(defaults read "$FRAMEWORK/Info.plist" CFBundleExecutable)
> FRAMEWORK_EXECUTABLE_PATH="$FRAMEWORK/$FRAMEWORK_EXECUTABLE_NAME"
> echo "Executable is $FRAMEWORK_EXECUTABLE_PATH"
> 
> EXTRACTED_ARCHS=()
> 
> for ARCH in $ARCHS
> do
> echo "Extracting $ARCH from $FRAMEWORK_EXECUTABLE_NAME"
> lipo -extract "$ARCH" "$FRAMEWORK_EXECUTABLE_PATH" -o "$FRAMEWORK_EXECUTABLE_PATH-$ARCH"
> EXTRACTED_ARCHS+=("$FRAMEWORK_EXECUTABLE_PATH-$ARCH")
> done
> 
> echo "Merging extracted architectures: ${ARCHS}"
> lipo -o "$FRAMEWORK_EXECUTABLE_PATH-merged" -create "${EXTRACTED_ARCHS[@]}"
> rm "${EXTRACTED_ARCHS[@]}"
> 
> echo "Replacing original executable with thinned version"
> rm "$FRAMEWORK_EXECUTABLE_PATH"
> mv "$FRAMEWORK_EXECUTABLE_PATH-merged" "$FRAMEWORK_EXECUTABLE_PATH"
> 
> done
> fi

Its Works. Thank u .
